parser/misc.go

```diff
@@ -146,6 +146,7 @@ var tokenMap = map[string]int{
 	"AFTER":                      after,
 	"ALL":                        all,
 	"ALTER":                      alter,
+	"ALWAYS":                     always,
```

Comment:
keyword should be add to `ReservedKeyword` or `UnReservedKeyword`, you can find it searching
`^ReservedKeyword` in parser.y , and add them to TestSimple in parser_test.go